src/Map/styled.js

```diff
@@ -29,7 +29,7 @@ export const LogoText = styled.p`
 export const StyledMap = styled.div`
   height: 100%;
   width: 100%;
-  position: ${p => p.fullScreen ? 'fixed' : 'relative'};
+  position: fixed;
```

Comment:
@UsmanAhmadSaeed this will be a breaking change- we still want the map to be able to take the shape of it's parent component when `fullScreen` is not true (to embed on a page instead of being the entire viewport for exmaple)

Reply:
i think i just realized...this was never a bug- it was working as expected. we just never set a `width`/`height` on the parent div so it was using `100%` of nothing and not showing up. When i hard set a width/height in the demo site it worked as expected (see screenshot): 
<img width="1680" alt="Screen Shot 2020-10-12 at 1 00 35 PM" src="https://user-images.githubusercontent.com/17538943/95777160-46a2e480-0c8b-11eb-991b-631f652a4a72.png">

Reply:
Alright, good to know that!